### PR TITLE
Feature : Dashboard Service and Dashboard Controller added

### PR DESCRIPTION
**Pull request by sagarxt**
#42 
-DashboardService class, DashboardServiceImpl, DashboardController added.

### CODE REVIEW

File: backend/src/main/java/com/moshop/backend/controller/DashboardController.java
Comment (smruti-banty) [inline backend/src/main/java/com/moshop/backend/controller/DashboardController.java:1]: Use service class as constructor inject not the Impl class

**Review by smruti-banty — COMMENTED** [review]
For now this logic is ok. But in future this may slow down the application. As it hits 5 times to db

File: backend/src/main/java/com/moshop/backend/controller/DashboardController.java
Comment (sagarxt) [reply backend/src/main/java/com/moshop/backend/controller/DashboardController.java:1]: Changes done as per comment

**Review by Abinashkjcjanscinanconanc — APPROVED** [review]
LGTM

**Pull request merged**